encoding/legacy-mb-japanese/iso-2022-jp/iso2022jp-encode-form-csiso2022jp.html.headers

```diff
@@ -0,0 +1 @@
+Content-Type: text/html; charset=csiso2022jp
```

Comment:
Can you add a trailing newline here and the other .headers files?

It looks like that was done for the shift_jis tests, and makes `cat`ing the files more pleasant